hivemind_bus_client/http_client.py

```
@@ -0,0 +1,388 @@
+            lang = message.metadata.get("lang")
+            utt = message.metadata.get("utterance")
+            file_name = message.metadata.get("file_name")
+            try:
+                self.bin_callbacks.handle_receive_tts(bin_data, utt, lang, file_name)
+            except:
+                LOG.exception("Error in binary callback: handle_receive_tts")
+        elif message.bin_type == HiveMindBinaryPayloadType.FILE:
+            file_name = message.metadata.get("file_name")
+            try:
+                self.bin_callbacks.handle_receive_file(bin_data, file_name)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)